solutions/standard/main.tf

```diff
@@ -0,0 +1,242 @@
+##############################################################################
+# IBM Cloud Provider
+##############################################################################
+
+provider "ibm" {
+  ibmcloud_api_key = var.ibmcloud_api_key
+  region           = var.region
+  ibmcloud_timeout = 60
+}
+
+##############################################################################
+
+########################################################################################################################
+# Landing Zone Locals
+########################################################################################################################
+
+locals {
+  out = replace(file("./override.json"), "mz2o-2x16", var.machine_type)
+  image = replace(local.out,"ibm-zos-2-5-s390x-dev-test-wazi-7", var.image_name)
+}
+
+##############################################################################
+
+##############################################################################
+# Landing Zone
+##############################################################################
+module "landing-zone" {
+  source                 = "terraform-ibm-modules/landing-zone/ibm//patterns//vsi//module"
+  version                = "5.18.0"
+  prefix                 = var.prefix
+  region                 = var.region
+  ssh_public_key         = var.ssh_public_key
+  override               = var.override
+  override_json_string   = local.image
+}
+
+########################################################################################################################
+# Resource Group
+########################################################################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.5"
+  #   if an existing resource group is not set (null) create a new one using prefix
+  existing_resource_group_name = "${var.prefix}-slz-management-rg"
+  depends_on                   = [module.landing-zone]
+}
+
+########################################################################################################################
+# Secrets Manager resources
+########################################################################################################################
+
+# Create a new SM instance if not using an existing one
+module "secrets_manager" {
+  source               = "terraform-ibm-modules/secrets-manager/ibm"
+  version              = "1.2.1"
+  resource_group_id    = module.resource_group.resource_group_id
+  region               = var.region
+  secrets_manager_name = "${var.prefix}-sm-instance"
+  sm_service_plan      = var.sm_service_plan
+  service_endpoints    = "public-and-private"
```

Comment:
In [v1.5.0](https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager/releases/tag/v1.5.0) of this module, `service_endpoints` has been updated to `allowed_network` and now only accepts the following values: `private-only` or `public-and-private`.
I suggest you update to the latest version (it is currently `v1.12.4`) but also you may wish to expose this as an input variable. I also think you may want to consider defaulting it to `private-only` so your solution is secure by default. Perhaps use `public-and-private` for quickstart version?